sql-statements/sql-statement-admin-recover.md

```diff
@@ -10,7 +10,7 @@ When the row data and index data are inconsistent, you can use the `ADMIN RECOVE
 ## Synopsis
 
 ```ebnf+diagram
-AdminCleanupStmt ::=
+AdminStmt ::=
     'ADMIN' 'RECOVER' 'INDEX' TableName IndexName
```

Comment:
```suggestion
    'ADMIN' 'RECOVER' 'INDEX' TableName Identifier
```

Reply:
If we use Identifier, shouldn't we also do that for the TableName?
```
    'ADMIN' 'RECOVER' 'INDEX' Identifier Identifier
```

We could define `IndexName` and we could define `Index` to consist of a table name and index name.
```
AdminStmt ::=
    'ADMIN' 'RECOVER' 'INDEX' Index

Index ::=
    TableName IndexName

TableName ::=
    Identifier

IndexName ::
    Identifier
```